scripts/create_irreducible_polys_database.py

```diff
@@ -0,0 +1,111 @@
+"""
+A script to create a database of irreducible polynomials
+
+Sources:
+ - Gadiel Seroussi. Table of Low-Weight Binary Irreducible Polynomials (1998): https://www.hpl.hp.com/techreports/98/HPL-98-135.html
+
+"""
+from __future__ import annotations
+
+import os
+import sqlite3
+from pathlib import Path
+
+import requests
+import hashlib
+from pdfminer.high_level import extract_text
+
+
+def main():
+    """
+    The main routine to create a database of irreducible polynomials
+    """
+
+    database_file = Path(__file__).parent.parent / "src" / "galois" / "_databases" / "irreducible_polys.db"
+    conn, cursor = create_database(database_file)
+
+    _add_hpl_1998(conn, cursor)
```

Comment:
Generalizing this to future additions, I think it's better to keep `main()` clean and deal with each source in its own function. What do you think?

Reply:
Yes, I totally agree!